modules/odc-customizing-available-cluster-roles-using-developer-perspective.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// applications/projects/working-with-projects.adoc
+
+[id="odc-customizing-available-cluster-roles-using-developer-perspective_{context}"]
+= Customizing the available cluster roles using the Developer perspective
+
+You can customize cluster role's name of existing projects by editing the YAML code in the `console.openshift.io/display-name: v1`.
+
+.Procedure
+Customize the cluster role options of all projects that are shown in the *Project Access* tab:
+
+. In the *Search* view, select the *Resources* drop down and search "*Console*".
+. Select the *Console `operator.openshift.io/v1`*.
++
+.Searching Console resource
+image::odc_cluster_console.png[]
```

Comment:
Though I find the screenshot helpful, I know they can be quite difficult for accessibility, a growing concern in RH documentation. They also are hard to maintain up to date. 

Personally, I find it best to avoid them. However, if your team for this specific document believes the screenshot would help all users, keep it.

Reply:
If you do keep the screenshot, end step two with a colon, not a period, since it introduces the screenshot:

2. Select the Console operator:

Reply:
In ODC we keep the sentence closed with a period. For reference check this page: https://docs.openshift.com/container-platform/4.9/applications/odc-viewing-application-composition-using-topology-view.html#odc-interacting-with-applications-and-components_viewing-application-composition-using-topology-view